src/components/navigation/document-tab-panel.tsx

```diff
@@ -142,33 +144,56 @@ export class DocumentTabPanel extends BaseComponent<IProps, IState> {
   private renderSubSections(subTab: any) {
     const { selectedDocument, onSelectNewDocument, onSelectDocument } = this.props;
     const { user } = this.stores;
+    const isInNetwork = user.type === "teacher" && user.network;
```

Comment:
You can just use `user.isNetworkedTeacher`.

Reply:
I migrated this code from another component, but I might as well fix it now.